requirements/requirements.txt

```diff
@@ -73,6 +73,7 @@ git+git://github.com/smartfile/django-transfer.git@6e0dc94c3341c358fca8eb2bf74e2
 Pygments==2.0.2
 tinys3==0.1.11
 django-formtools==2.0
+django-otp==0.3.13
```

Comment:
this is up from 0.3.7 on mine. That seems fine / intended, but just checking—the point here is to upgrade to a (point-release) version that does not suffer from these deprecation warnings?